dep-graph/dep-graph/src/app/mock-project-graph-service.ts

```diff
@@ -33,7 +33,7 @@ export class MockProjectGraphService implements ProjectGraphService {
         {
           source: 'existing-app-1',
           target: 'existing-lib-1',
-          type: 'statis',
+          type: 'static' as any,
```

Comment:
I narrowed the type of the property `ProjectGraphDependency['type']` from `DependencyType | string` to simply `DependencyType` as it is now being used a little bit more and for the most part the narrower type is helpful. However, for some reason, in this module, importing the enum `DependencyType` from `@nrwl/devkit` was causing an extremely bizarre error in the `dep-graph-dep-graph:build-base` task, which I captured here:

![image](https://user-images.githubusercontent.com/39352647/146186024-54e5b8ea-249b-4648-86fe-a026c9bf44a4.png)

What makes it even more bizarre is that there are a number of other modules in this same app that are importing the enum `DependencyType` without any issue. My only guess is that the difference here is the presence of the decorator, but why that would cause the above error is beyond me.

At any rate, adding `as any` seemed like a decent compromise.